### PR TITLE
Fetch setup for site root if not present (for symfony routes)

### LINKED ISSUE #85: Force TemplateService initialization
When dealing with Symfony routes the TYPO3 FrontendController is loaded from cache and therefore the TemplateService property isn't initialized correctly. This results in an empty TemplateService configuration which may be required by a service (e.g. reading Solr TypoScript configuration).

### PR DESCRIPTION
**Pull request by jdkpix**
| Q | A
| --- | ---
| Fixed tickets | fixes #85
| Related issues/PRs | -
| License | GPL-3.0+

#### What's in this PR?
Get the setup of the site root by default

### CODE REVIEW

File: EventSubscriber/FrontendControllerSubscriber.php [hunk 1]
```diff
@@ -82,6 +83,12 @@ public function onKernelRequest(GetResponseEvent $event): void
                 $frontendController->settingLanguage();
                 $frontendController->settingLocale();
             }
+
+            if (!$frontendController->tmpl->setup && ($site = $event->getRequest()->attributes->get('site')) instanceof SiteInterface) {
```
Comment (EmiiKhaos) [inline EventSubscriber/FrontendControllerSubscriber.php]: Check with count if array is not empty. Assigned outside of if.

File: EventSubscriber/FrontendControllerSubscriber.php [hunk 2]
```diff
@@ -82,6 +83,12 @@ public function onKernelRequest(GetResponseEvent $event): void
...
+                $frontendController->id = $site->getRootPageId();
+                $frontendController->determineId();
+                $frontendController->getConfigArray();
+            }
```
Comment (EmiiKhaos) [inline EventSubscriber/FrontendControllerSubscriber.php]: Is the order correct, to determine it after language or should it be moved up?

File: EventSubscriber/FrontendControllerSubscriber.php (hunk 1 above)
Comment (jdkpix) [reply EventSubscriber/FrontendControllerSubscriber.php]: Okay

File: EventSubscriber/FrontendControllerSubscriber.php (hunk 2 above)
Comment (jdkpix) [reply EventSubscriber/FrontendControllerSubscriber.php]: You are right, initialization oder is different in typo3 middlewares. Fixed it.

**Comment by EmiiKhaos** [comment]
Please, amend your commits if they are simply code style changes like that.

And I pushed the missing line in the changelog

**Comment by EmiiKhaos** [comment]
Released as 2.0.3

**Pull request merged**